src/main/resources/META-INF/sitemap.tld

```diff
@@ -49,9 +49,14 @@
         <function-signature>boolean excludeNode(org.jahia.services.content.JCRNodeWrapper,javax.jcr.NodeIterator)</function-signature>
     </function>
     <function>
-    <name>getGuestNodes</name>
-    <function-class>org.jahia.modules.sitemap.utils.QueryHelper</function-class>
-    <function-signature>java.util.Set getGuestNodes(java.lang.String,java.lang.String)</function-signature>
-</function>
+        <name>getGuestNodes</name>
```

Comment:
what is this function for? not seeing it used anywhere. should we remove?

Reply:
I am not sure as it is there already